Remotion/Web/Development.WebTesting/Schemas/WebTestingConfiguration.xsd

```diff
@@ -21,6 +21,11 @@
             <xs:attribute name="disableSecurityWarningsBehavior" type="c:ChromiumDisableSecurityWarningsBehavior" default="Ignore" />
           </xs:complexType>
         </xs:element>
+        <xs:element name="edge" minOccurs="0" maxOccurs="1">

```

Comment:
Refactor into separate xml type to allow reuse as both the edge and the chrome element.